adapters/handlers/rest/raft/handler.go

```diff
@@ -164,6 +176,7 @@ func ClusterRouter(schemaHandler schema.Handler) *http.ServeMux {
 	r.HandleFunc(root+"/join", raftHandler.JoinNode)
 	r.HandleFunc(root+"/remove", raftHandler.RemoveNode)
 	r.HandleFunc(root+"/statistics", raftHandler.Statistics)
+	r.HandleFunc(root+"/revert", raftHandler.RestoreSchemaToV0)
```

Comment:
Using 'revert' might be misleading since we're not reverting; we're simply storing a copy of the schema in the old format.

You can try something like  `/schema/v1 `with POST as the HTTP method

Reply:
yeah i have been back and force but will be at the ned `/v1/cluster/schema/v1` it's a bit off naming but i can do it of what we want